3/neo4j.js

```diff
@@ -0,0 +1,16 @@
+const neo4j = require('neo4j-driver');
+
+const uri = "bolt://localhost:7687"
+const user = "neo4j"
+const password = "graphdb"
```

Comment:
You just leaked all your secrets, now Imma steal your girl! 😋 
Suggestion: Using `require('dotenv').config();` to retrieve all your environment properties saved in an `.env` file and read them with `process.env`.

Reply:
Also difficult for me if I want to change the credentials. This should go into a `.gitignore`d file.